lib/__tests__/findMatchingFiles-test.js

```diff
@@ -0,0 +1,108 @@
+import childProcess from 'child_process';
+import fs from 'fs';
+import path from 'path';
+
+import mkdirp from 'mkdirp';
+
+import findMatchingFiles from '../findMatchingFiles';
+
+describe('findMatchingFiles', () => {
+  const tmpDir = './tmp-findMatchingFiles';
+  let subject;
+  let word;
+  let existingFiles;
+  let lookupPath;
+
+  beforeEach(() => {
+    fs.mkdirSync(tmpDir);
+    word = 'foo';
+    existingFiles = [];
+    lookupPath = tmpDir;
+
+    subject = () => {
+      existingFiles.forEach((file) => {
+        const fullPath = `${tmpDir}/${file}`;
+        mkdirp.sync(path.dirname(fullPath));
+        fs.closeSync(fs.openSync(fullPath, 'w')); // create empty file
+      });
+
+      return findMatchingFiles(lookupPath, word);
+    };
+  });
+
+  afterEach(() => {
+    childProcess.execSync(`rm -r ${tmpDir}`);
```

Comment:
What do you think about using rimraf instead of this? That should give us windows compatibility, for instance. https://github.com/isaacs/rimraf